en/config_mc/pid_tuning_guide_multicopter.md

```diff
@@ -49,14 +51,50 @@ Here are some general points to follow when tuning:
+> **Note** A well-tuned rate controller is very important as it affects *all* flight modes.
+  A badly tuned rate controller will be visible in [Position mode](../flight_modes/position_mc.md), for example, as "twitches" (the vehicle will not hold perfectly still in the air).
 
+#### Rate Controller Architecture/Form
 
-> **Note** A well-tuned rate controller is very important as it affects *all* flight modes. A badly tuned rate controller will be visible in [Position mode](../flight_modes/position_mc.md), for example, as "twitches" (the vehicle will not hold perfectly still in the air).
+PX4 supports two (mathematically equivalent) forms of the PID rate controller in a single mixed implementation: Parallel and Standard.
+
+![PID_Mixed](../../images/mc_pid_tuning/PID_algorithm_Mixed.png)
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1hXnAJVRyqNAdcreqNa5W4PQFkYnzwgOO/view?usp=sharing -->
+- _G(s)_ represents the angular rates dynamics of a vehicle
+- _r_ is the rate setpoint
+- _y_ is the body angular rate (measured by a gyro)
+- _e_ is the error between the rate setpoint and the measured rate
+- _u_ is the output of the PID controller
+
+> **Note** The derivative term (**D**) is on the feedback path in order to avoid an effect known as the [derivative kick](http://brettbeauregard.com/blog/2011/04/improving-the-beginner%E2%80%99s-pid-derivative-kick/).
+
+Users can select the form that is used by setting the proportional gain for one of the forms  (**K** or **P**) to "1".
+The resulting forms are described below.
+
+##### Parallel Form
+
+This form is commonly used in textbooks as this is the simplest form where the output of the controller is simply the sum of the proportional, integral and derivative actions.
+
+  ![PID_Parallel](../../images/mc_pid_tuning/PID_algorithm_Parallel.png)
+
+##### Standard Form
+This form is mathematically equivalent to the parallel form but facilitates manual tuning as the K gain represents the global "aggressiveness" of the controller. The integral gain is the inverse of the integrator time constant and represents how fast the integrator grows compared to the proportional gain. Similarly, the derivative gain is the derivative time constant and represents how much damping is added to the system.
+
+
+  ![PID_Standard](../../images/mc_pid_tuning/PID_algorithm_Standard.png)
+
+References:
+- https://www.controleng.com/articles/not-all-pid-controllers-are-the-same/
+- https://en.wikipedia.org/wiki/PID_controller#Standard_versus_parallel_(ideal)_PID_form
+
+#### Rate PID Tuning
+
+The related parameters for the tuning of the PID rate controllers are:
+- Roll rate control ([MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P), [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I), [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D), [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K))
```

Comment:
@bresch
1.  I believe from all this that you EITHER tune in the standard or in the parallel form - correct? 
    - So you would start by setting `MC_ROLLRATE_P` OR `MC_ROLLRATE_K` to 1 (but not both)? (That is implied but not stated).
    - If so, then IMO the P and D params should be grouped here to make it obvious.

Reply:
> I believe from all this that you EITHER tune in the standard or in the parallel form - correct? 

Correct

> Is it correct that can MC_ROLLRATE_K can have a max value of 5, but MC_ROLLRATE_P can only have max of 0.5 (i.e. just making sure that this is reasonable and not a typo).

Thanks for rising that. I'll change the `MC_(ROLL|PITCH|YAW)RATE_P` maximum value to 1